test/unit/spec/appWidgets/validationSpec.js

```diff
@@ -1127,6 +1093,102 @@ define([
             });
         });
 
+        // validateBoolean
```

Comment:
moved this up from further down in the file so that the workspace-related tests were kept together